app/views/tags/show.html.erb

```diff
@@ -24,12 +24,13 @@
   <!--main content-->
   <p><%= ts("This tag belongs to the %{tag_class} Category.", :tag_class => @tag.class::NAME) %>
   <% if @tag.canonical %>
+    <%= t(".canonical_html",
+          canonical_tag_faq_link: link_to(t(".canonical_tag_faq"), archive_faq_path("glossary", anchor: "canonicaldef")),
+          filter_works_link: link_to(t(".filter_works"), { controller: :works, action: :index, tag_id: @tag }),
+          filter_bookmarks_link: link_to(t(".filter_bookmarks"), { controller: :bookmarks, action: :index, tag_id: @tag })) %>
```

Comment:
Do `tag_bookmarks_path(@tag)` and `tag_works_path(@tag)` work here?

Reply:
Yep, changed.